package.json

```diff
@@ -14,7 +14,7 @@
     "build:dist": "./node_modules/.bin/webpack --config webpack.config.js --mode=development",
     "build:dist:dev": "./node_modules/.bin/webpack --config webpack.dev.config.js --mode=development",
     "build:dist:prod": "./node_modules/.bin/webpack --config webpack.config.js --mode=production --output-filename threeid-provider.min.js",
-    "build": "npm run build:es5; npm run build:dist; npm run build:dist:prod",
```

Comment:
Shall we remove the `build:dist` scripts, webpack configs and dependencies as well or part of another PR maybe?